bundler/spec/other/major_deprecation_spec.rb

```diff
@@ -574,10 +574,6 @@
         bundle :show
       end
 
-      it "prints a deprecation warning recommending `bundle list`", :bundler => "< 3" do
-        expect(deprecations).to include("use `bundle list` instead of `bundle show`")
-      end
-
```

Comment:
For the specs being removed, they're usually following by a pending "fails with a helpful" message spec, that should've been implemented before we release bundler 3. Since these specs won't be implemented anymore, they should be removed too.